_posts/2020-01-20-sentry-the-guardian-of-your-quarkus-app.adoc

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: 'Sentry Error Tracker - A guardian of your Quarkus application'
 date: 2020-01-20
-tags: quarkus extension logging cloud error
+tags: extension sentry tracing
```

Comment:
sentry?

Reply:
applying the "tag for main technology the extension blog is about" ...

Reply:
I always come back to, will we want to filter blogs by sentry refs?

Reply:
right now? probably not so much - over time..yeah...similar to "hibernate", "kafka", etc. wuold be interesting for those with tech specific interest.